lms/templates/main.html

```diff
@@ -137,7 +137,7 @@
     % endif
 
     <div class="content-wrapper" id="content">
-      ${self.body()}
+      ${next.body()}
```

Comment:
Right -- see here for more info:  http://docs.makotemplates.org/en/latest/inheritance.html#using-the-next-namespace-to-produce-content-wrapping